src/renderer/services/deposit/DepositTransactionService.ts

```diff
@@ -0,0 +1,83 @@
+import {Keypair as KeyPair} from "@chainsafe/bls/lib/keypair";
+import bls from "@chainsafe/bls";
+import {BLSPubkey as BLSPubKey} from "@chainsafe/eth2.0-types";
+import {createHash} from "crypto";
+import {DepositData} from "@chainsafe/eth2.0-types/lib/misc";
+import BN from "bn.js";
+import {signingRoot} from "@chainsafe/ssz";
+import {BLSDomain} from "@chainsafe/bls/lib/types";
+import abi from "ethereumjs-abi";
+import {config} from "@chainsafe/eth2.0-config/lib/presets/mainnet";
+import Units from "ethereumjs-units";
+import {IDepositParams, ITx} from "./DepositTransactionTypes";
+
+// fixed deposit amount
+const depositAmountInEth = "32";
+
+// fixed BLS domain -> deposit = 3
+const depositBLSDomain: BLSDomain =
+    Buffer.concat([new BN(3).toArrayLike(Buffer, "le", 4), Buffer.alloc(4)]);
+
+// definition of contracts deposit function
+const depositFunctionSignature = "deposit(bytes,bytes,bytes,bytes32)";
+
+// deposit contract address
+const depositContractAddress = "0x9c86825280b1d6c7dB043D4CC86E1549990149f9";
```

Comment:
@mpetrunic I defined const for deposit contract address, not sure if it fine for now?

Reply:
it should be param, we will have local/testnet/mainnet all different

Reply:
@mpetrunic function param or env variable ?